src/pages/my_page/user_profile/MyResults.tsx

```diff
@@ -32,6 +34,15 @@ export const MyResults = () => {
     rowsPerPage * (page - 1)
   );
 
+  const promotedPublicationsQuery = useQuery({
+    queryKey: ['person-preferences', personId],
```

Comment:
Ser at `personId` kan være tom streng, så kanskje sikre at den har en verdi før vi prøver å hente personens preferanser?
```suggestion
  const promotedPublicationsQuery = useQuery(
    enabled: !!personId,
    queryKey: ['person-preferences', personId],
```